app/utilities.py

```diff
@@ -1,6 +1,18 @@
 import base64
 from io import BytesIO
+from app.manifest_name_to_discover_name import name_map
 
+def check_path_in_mangled_list(s3_path):
```

Comment:
This method is poorly named it is not checking that the given path is in the mangled list.  This method is returning a mangled name.

Reply:
maybe get_path_from_mangled_list or find_path_in_mangled_list